src/network/networkd-dhcp6.c

```diff
@@ -835,7 +832,44 @@ static int dhcp6_set_unreachable_route(Link *link, const union in_addr_union *ad
 
         (void) set_remove(link->dhcp6_routes_old, ret);
 
-        return 1;
+        return 0;
+}
+
+static int log_dhcp6_pd_prefix(Link *link, const union in_addr_union *prefix, uint8_t prefixlen) {
```

Comment:
I can't say I like this name: log… implies that this will, well, just log. I think the way that this is structured is now confusing.

Reply:
OK, renamed to `dhcp6_pd_prefix_add()`.